libs/net/dal/Services/ReportService.cs

```diff
@@ -105,6 +105,7 @@ public IEnumerable<Report> Find(ReportFilter filter)
             .Include(r => r.Owner)
             .Include(r => r.Template).ThenInclude(t => t!.ChartTemplates)
             .Include(r => r.Sections).ThenInclude(s => s.Filter)
+            .Include(r => r.Instances)
```

Comment:
This needs to be removed.  Think about what this does.  It returns every instance generated for the report.  After a year it's 365 items in an array, of which at best only one will ever be viewed.  I'll remove it from my next PR.